pkgs/development/misc/cppreference-doc/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, fetchzip }:
+
+let
+  pname = "cppreference-doc";
+  version = "2022.07.30";
+  ver = builtins.replaceStrings ["."] [""] version;
+
+in fetchzip {
+  name = pname + "-" + version;
+
+  url = "https://github.com/PeterFeicht/${pname}/releases/download/v${ver}/html-book-${ver}.tar.xz";
+  sha256 = "sha256-gsYNpdxbWnmwcC9IJV1g+e0/s4Hoo5ig1MGoYPIHspw=";
+
+  stripRoot = false;
+
+  postFetch = ''
+    rm $out/cppreference-doxygen-local.tag.xml $out/cppreference-doxygen-web.tag.xml
+    mkdir -p $out/share/cppreference/doc
+    mv $out/reference $out/share/cppreference/doc/html
+  '';
+
+  passthru = { inherit pname version; };
+
+  meta = with lib; {
+    description = "C++ standard library reference";
+    homepage = "https://en.cppreference.com";
+    license = licenses.cc-by-sa-30;
+    maintainers = with maintainers; [ panicgh ];
+    platforms = platforms.all;
```

I've seen in other cases where `postFetch` was used, that `meta.hydraPlatforms = [];` was set.

This makes me wonder if Hydra should build this pkg, which is only downloading and unpacking?

```suggestion
    platforms = platforms.all;
    hydraPlatforms = [];
```

---

Normally we set this if the license does not allow redistributing via hydra or the derivation is to big or it makes no sense to cache it. I think it is fine to keep it here.